pkg/plugin/standalone/registry_test.go

```diff
@@ -63,33 +63,33 @@ func testPluginBlueprint() blueprint {
 			SourceParams: map[string]plugin.Parameter{
 				testPluginSourceParam1: {
 					Default:     testPluginSourceParam1Default,
-					Type:        "string",
+					Type:        plugin.ParameterTypeString,
 					Description: testPluginSourceParam1Description,
 					Validations: []plugin.Validation{
 						{Type: plugin.ValidationTypeRequired, Value: ""},
 					},
 				},
 				testPluginSourceParam2: {
 					Default:     testPluginSourceParam2Default,
-					Type:        "string",
+					Type:        plugin.ParameterTypeString,
 					Description: testPluginSourceParam2Description,
-					Validations: nil,
+					Validations: []plugin.Validation{},
```

Comment:
It would be great to adjust the test to see if validations are also sent through. We should add some validations in the [test plugin](https://github.com/ConduitIO/conduit/blob/maha/parse-validations/pkg/plugin/standalone/test/testplugin/main.go).